kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/cdi/ProcessRequestBean.java

```diff
@@ -325,11 +325,11 @@ private Context getDeploymentCommandContext(List<String> correlationKeyProps, Lo
         // process instance
         if( processInstanceId == null && cmd instanceof ProcessInstanceIdCommand ) {
             processInstanceId = ((ProcessInstanceIdCommand) cmd).getProcessInstanceId();
-            if( processInstanceId != null ) {
+            if (processInstanceId != null && processInstanceId != -1L) {
```

Comment:
I think there is unnecessary duplicate checking, it can be reformulated as:
`     if( processInstanceId == null && cmd instanceof ProcessInstanceIdCommand ) {
            processInstanceId = ((ProcessInstanceIdCommand) cmd).getProcessInstanceId();
        }
        if (processInstanceId != null && processInstanceId != -1L) {
            return ProcessInstanceIdContext.get(processInstanceId);
        }
`